tests/ci/aml_cpu_unit_tests.yml

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+variables:
+  - group: AML-config
+  - name: "test"
+    value: "tests/ci/run_pytest.py"
+  - name: "testfolder"
+    value: "./tests/unit"
+  - name: "testmarkers"
+    value: '"not notebooks and not gpu and not azureml"'
+  - name: "junitxml"
+    value: "reports/test-unit.xml"
+  - name: "maxnodes"
+    value: 4
+  - name: "reponame"
+    value: "nlp"
+  - name: "clustername"
+    #underscores not allowed, only letters, digits and - char
+    value: "nlp-cpu"
+  - name: "vmsize"
+    value: "STANDARD_D3_V2"
+  - name: "dockerproc"
+    value: "cpu"
+  - name: "expname"
+    value: "cpu_unit_tests"
+
+trigger: none
+
+pr:
+  - staging
+  - master
+
+jobs:
+  - job: "Unit_test_cpu"
+    timeoutInMinutes: 120
+
+    pool:
+      vmImage: "ubuntu-16.04"
+
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.6"
+          architecture: "x64"
+        displayName: "Use Python 3.6"
+
+      - script: |
+          az login --service-principal -u $(spidentity) -p $(spsecret) --tenant $(sptenant)
+        displayName: "Login to Azure"
+
+      - bash: |
+          echo "##vso[task.prependpath]/data/anaconda/bin"
+        displayName: Add Conda to PATH
+
+      - script: pip install paramiko==2.4.2 &&
+          pip install azureml-sdk &&
+          pip install azure-cli
+        displayName: "install azureml-sdk"
+
+      - script: python tools/generate_conda_file.py --name nlp
+        displayName: " generate_conda_file.py"
+
+      - script: python tests/ci/submit_azureml_pytest.py --subid $(SubscriptionID) --testfolder $(testfolder) --testmarkers $(testmarkers) --clustername $(clustername) --expname $(expname) --dockerproc $(dockerproc) --junitxml $(junitxml) --reponame $(reponame) --branch $(Build.SourceBranchName)
```

Comment:
could you please add here `--condafile` explicitely? When seeing this it was confusing to me that the conda file is being generated and used in the submit_azureml_pytest.py with its default value